<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SPRSettingsIMSWidget</class>
 <widget class="QWidget" name="SPRSettingsIMSWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>778</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="15,0">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>180</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Настройка работы</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignHCenter">
          <widget class="QTableWidget" name="tMinMaxParams">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="autoScroll">
            <bool>true</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>150</number>
           </attribute>
           <attribute name="verticalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
           <row>
            <property name="text">
             <string>Время измерения куска (мс)</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Время задержки на срабатывания ИМ (мс)</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Длительность импульса ИМ (мс)</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Мин. кусок</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Макс. кусок</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Макс. камень</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string>16</string>
            </property>
           </item>
           <item row="0" column="1">
            <property name="text">
             <string>100</string>
            </property>
           </item>
           <item row="0" column="2">
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item row="1" column="0">
            <property name="text">
             <string>150</string>
            </property>
           </item>
           <item row="1" column="1">
            <property name="text">
             <string>100</string>
            </property>
           </item>
           <item row="1" column="2">
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item row="2" column="0">
            <property name="text">
             <string>40</string>
            </property>
           </item>
           <item row="2" column="1">
            <property name="text">
             <string>80</string>
            </property>
           </item>
           <item row="2" column="2">
            <property name="text">
             <string>0</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_6" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>Временные характеристики работы ИМ</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QwtPlot" name="plotParams">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="autoReplot">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_7" native="true">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>120</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="widget" native="true">
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <property name="spacing">
                   <number>3</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Параметр блокировки ИМ (срабатываний в секунду)</string>
                    </property>
                    <property name="wordWrap">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="leBlockParam">
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>10</number>
                    </property>
                    <property name="value">
                     <number>8</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QWidget" name="widget_2" native="true">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <property name="spacing">
                   <number>3</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QWidget" name="widget_5" native="true">
                    <layout class="QHBoxLayout" name="horizontalLayout">
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gbStatistic">
           <property name="title">
            <string>История работы ИМ</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QwtPlot" name="plotStatistuka">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="wStatistic" native="true">
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="3">
                <widget class="QCheckBox" name="cbTh3">
                 <property name="font">
                  <font>
                   <pointsize>8</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Ручей 4</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QCheckBox" name="cbTh2">
                 <property name="font">
                  <font>
                   <pointsize>8</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Ручей 3</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QCheckBox" name="cbTh0">
                 <property name="font">
                  <font>
                   <pointsize>8</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Ручей 1</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QCheckBox" name="cbTh1">
                 <property name="font">
                  <font>
                   <pointsize>8</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Ручей 2</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" colspan="2">
                <widget class="QPushButton" name="bClearStatistic">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Сбросить счетчики</string>
                 </property>
                 <property name="icon">
                  <iconset resource="icons.qrc">
                   <normaloff>:/icons/icons/edit-clear.png</normaloff>:/icons/icons/edit-clear.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_9" native="true">
              <property name="font">
               <font>
                <family>Tahoma</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="spacing">
                <number>3</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Макс. количество срабатываний</string>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="leLimitAbrasion">
                 <property name="maximum">
                  <number>10000000</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
